receiver/mongodbreceiver/metadata.yaml

```
@@ -324,3 +324,22 @@ metrics:
       aggregation: cumulative
       monotonic: true
     attributes: [database, lock_type, lock_mode]
+  mongodb.health:
+    enabled: false
+    description: The health status of mongodb.
```

```suggestion
    description: The health status of the server.
```

Or is this per node? Either way, let's be as specific as possible here.

---

should be the server